packages/client-core/src/user/UserUISystem.tsx

```diff
@@ -91,7 +93,7 @@ const reactor = () => {
       [UserMenus.Profile]: ProfileMenu,
       [UserMenus.Settings]: SettingMenu,
       [UserMenus.Settings2]: SettingMenu2,
-      [UserMenus.AvatarSelect]: AvatarSelectMenu,
```

Comment:
`AvatarSelectMenu` will not be used elsewhere. Hence we can remove the file altogether.
Thus, we can rename `AvatarSelectMenu2` to `AvatarSelectMenu`

Reply:
I was told by @HexaField to leave the current files with MUI as they are and add new files for the new components with tailwind.

Reply:
Yes, we still need the old MUI UI for locations until we overhaul the whole location UI. It needs to look correct together.

Reply:
Though this is half of the work of the UI screen, so it won't be much more to update the other UIs. Great work.